docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -331,6 +347,93 @@ The example below defines the following table access policy:
       ]
     }
 
+.. _system-file-function-rules:
+
+Function rules
+^^^^^^^^^^^^^^
+
+These rules control the ability of a user to execute SQL functions, including
+PTFs (Polymorphic Table Functions).
```

Comment:
```suggestion
These rules control the user's ability to execute SQL functions.
```
Same thing as above where I don't think we need to say functions _and_ PTFs.

Reply:
Mixed with https://github.com/trinodb/trino/pull/14774#discussion_r1009696805